src/references.d.ts

```diff
@@ -1,11 +1,11 @@
-/// <reference path="./typings/mdc.android.d.ts" />
-/// <reference path="./typings/mdc.ios.d.ts" />
+// /// <reference path="./typings/mdc.android.d.ts" />
+// /// <reference path="./typings/mdc.ios.d.ts" />
 
-interface MDCAlertController {
-    alertView: MDCAlertControllerView;
-    mdc_dialogPresentationController: MDCDialogPresentationController
-}
-interface MDCAlertControllerView extends UIView {
-    customContentView: UIView;
-    calculateContentSizeThatFitsWidth(width: number): CGSize;
-}
+// interface MDCAlertController {
```

Comment:
I think it was moved out, in order to have both declarations working together, the `mdc.android.d.ts` and `mdc.ios.d.ts` with the platform ones, because we are using the version `1.1.0-alpha07` for material components and the {N} team is using the version `1.0.0` and the declarations files that they generated are for that version.

Reply:
@henrychavez this will be fixed by the {N} team. Already talked about this with them.
It work as it was. If not please share the error